api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/support/CasConfigurationJasyptCipherExecutor.java

```diff
@@ -47,7 +52,11 @@ public CasConfigurationJasyptCipherExecutor(final Environment environment) {
         setProviderName(pName);
         val iter = getJasyptParamFromEnv(environment, JasyptEncryptionParameters.ITERATIONS);
         setKeyObtentionIterations(iter);
-        setInitializationVector();
+        val initializationVector = Boolean.parseBoolean(
+                getJasyptParamFromEnv(environment, JasyptEncryptionParameters.INITIALIZATION_VECTOR));
+        if (initializationVector || requiresInitializationVector(alg)) {
+            setInitializationVector();
```

Comment:
Rename to `configureInitializationVendor()`. Set implies that you're passing a value as a setter.